nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/resource/ExistingResource.java

```diff
@@ -353,4 +411,12 @@ protected Date getTimePropertyWrapper(DocumentModel doc, String name) {
         }
     }
 
+    protected DocumentModel markForBackMove(DocumentModel doc) {
+        doc.setPropertyValue(DC_SOURCE, doc.getPathAsString());
```

Comment:
Please only store the original name in the dc:source, not the full path (which is redundant information and slightly dangerous to use with `new PathRef`)